dotnet/src/SemanticKernel.Skills/Skills.UnitTests/Grpc/Protobuf/TestSkills/ResourceSkillsProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Resources;
+
+namespace SemanticKernel.Skills.UnitTests.Grpc.Protobuf.TestSkills;
+
+internal static class ResourceSkillsProvider
+{
+    /// <summary>
+    /// Loads OpenApi document from assembly resource.
```

Comment:
Is this  wording accurate?

Reply:
Nope, it's not, it's a copy-pasting issue. Fixed.